card/mod/standard/set/abstract/wql_search.rb

```diff
@@ -29,7 +29,7 @@ def fetch_query args={}
 end
 
 def standardized_query_args args
-  args.symbolize_keys!
+  args = args.symbolize_keys
```

Comment:
yes.  in some cases, the args that makes it here is not a normal Hash but a Card::Controller::Parameters (or something) object that does not respond to symbolize_keys! (with the bang).

not sure why it only shows up occasionally, but I've seen it, which is why I originally made the change.